spec/nero/input/kafka_reader_spec.rb

```diff
@@ -25,4 +25,17 @@
     expect(processor).to have_received(:process).with("id" => 2)
     expect(processor).to have_received(:process).with("id" => 3)
   end
+
+  it 'processes messages in the expected order' do
+    producer = Poseidon::Producer.new(brokers, "test_producer", partitioner: ->(partition, key) { 0 })
```

Comment:
Unused block argument - `partition`. If it's necessary, use `_` or `_partition` as an argument name to indicate that it won't be used. Also consider using a proc without arguments instead of a lambda if you want it to accept any arguments but don't care about them.<br>Unused block argument - `key`. If it's necessary, use `_` or `_key` as an argument name to indicate that it won't be used. Also consider using a proc without arguments instead of a lambda if you want it to accept any arguments but don't care about them.